code/modules/cargo/packs.dm

```diff
@@ -1037,13 +1094,6 @@
 	contains = list(/obj/item/stack/sheet/metal/fifty)
 	crate_name = "metal sheets crate"
 
-/datum/supply_pack/materials/plasteel20
-	name = "20 Plasteel Sheets"
-	desc = "Reinforce the station's integrity with twenty plasteel sheets!"
-	cost = 7500
-	contains = list(/obj/item/stack/sheet/plasteel/twenty)
-	crate_name = "plasteel sheets crate"
-
 /datum/supply_pack/materials/plasteel50
```

Comment:
Move this up to where the other plasteel entry is for organization conformity

Reply:
It being there *is* for organization conformity